script.js

```diff
@@ -1,4 +1,215 @@
+var gameMode = "Waiting for Number of Players";
+var gameStart = false;
+var playerScores = [];
+const winnerList = [];
+var noOfPlayers = 0;
+var currentPlayer = 0;
+var playerDiceRolls = [];
+var previousScores = []; //counter for round wins
+var previousWinners = []; // counter for winners each round
+var gamePlayMode = "Choose Play Mode";
+var noOfDice = 2;
```

Comment:
Great use of global variables at the top of the script.js file. For consistency, declare your variables with `var`. You might also want to set your initial currentPlayer to 1, and in your code below to increment that currentPlayer variable **after** you're done applying your logic, as opposed to before.